app/views/admin/layouts/edit.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= javascript_include_tag('https://unpkg.com/interactjs@1.3.3/dist/interact.min.js') %>
+
+<div class="container">
+  <div class="row">
+    <div class="col-sm-12">
+      <h1>Layout for <%= @grouping_name %></h1>
+    </div>
+  </div>
+  <div class="row">
+    <div class="layout-area col-sm-12">
+        <%= render partial: 'area', collection: @desks %>
+        <%= render partial: 'area', collection: @child_groupings %>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-sm-12">
+      <%= form_for @grouping, url: admin_layout_path, html: {method: 'put'} do |f| %>
```

Comment:
You can bring the method out of the `html` key, as in `url: admin_layout_path, method: :put`